web_server.js

```diff
@@ -56,3 +56,7 @@ io.on('connection', function (socket) {
 server.listen(process.env.PORT || 80, () => {
 	log.info(`Server is listening on port: ${server.address().port}`);
 });
+
+server.on('error', error => {
+	log.error(`Error while starting the server : ${error.message}`);
```

Comment:
Please log the whole error and not just the message and remove the space in front of the colon.

Reply:
Another thing: please move the error listener above the `server.listen` call.

Reply:
OK